Write "looking ahead" sections for all chapters (#511)
worldbank/dime-data-handbook: chapters/7-publication.tex

as their access is more restrictive,
and organizations limit access to platforms other than the one officially adopted.

\section{Looking ahead}

> This chapter described a the process that brings all your efforts together --

Reviewer: ```suggestion
This chapter described the culmination of all the efforts in data acquisition, cleaning, processing, and analyzing --
```

the production of materials to share with the world

> that combine design, data, processing, and analysis
> to answer a scientific or policy question in a way that has 

Reviewer: ```suggestion
that answer a scientific or policy question in a way that has 
```

never been done before.
Making sure that everything you discovered and created
is as broadly available and as easy to use as possible

> is the last step in producing scientific work from original data.

Reviewer: ```suggestion
is the last step in producing scientific evidence from original data.
```

This is the purpose of all the rigor, organization, and documentation
we encourage and detail at every step of the process:
since all your research materials are 
continuously organized, shareable, secure, documented, and readable,
they are both valuable to you and others and easy to provide access to.

> From here on, what to do with that is your decision.

Reviewer: This seems like a strange last word ... i would just leave it with the previous sentence

Author: ```suggestion
```
